cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc

```diff
@@ -12,6 +12,19 @@ If you import a VMware template, the wizard creates a virtual machine based on t
 include::modules/technology-preview.adoc[leveloffset=+1]
 :!FeatureName:
 
+.Prerequisites
+
+* You must create a VDDK image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
```

Comment:
You should move this prereq back to the procedure module. Since you do this as part of the assembly, it doesn't make sense as an assembly prereq.

Reply:
I was wondering whether I should try to keep the prerequisites together (in the procedure module) or put them in the assembly so that I could add the link to the kbase article. I'll try splitting them. Thanks.

Reply:
Yeah, you can split them up! I think it's better for the prereqs to go as close to the relevant content as possible whenever you can. (Some of the new portal wireframes have individual modules published separately, so it will eventually be possible to retrieve just that procedure module.)

Reply:
Good to know! I made the changes. Thanks!